packages/video_player/video_player/lib/video_player.dart

```diff
@@ -207,7 +207,10 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// null. The [package] argument must be non-null when the asset comes from a
   /// package and null otherwise.
   VideoPlayerController.asset(this.dataSource,
-      {this.package, this.closedCaptionFile, this.videoPlayerOptions})
+      {this.package,
+      this.closedCaptionFile,
+      this.videoPlayerOptions,
+      this.updateDuration = const Duration(milliseconds: 500)})
```

Comment:
should this be `valueNotifierInterval` or `updateInterval` or something like that?